improver_tests/spotdata/test_SpotExtraction.py

```diff
@@ -105,36 +101,28 @@ def setUp(self):
             anchor_start=False,
         )
 
-        times = np.array([time + timedelta(hours=i) for i in range(-3, 2)])
+        locations = np.array([0 + i for i in range(-3, 2)])
 
         # Create as int64 values
-        time_points = np.array([_create_time_point(time) for time in times])
-        bounds = np.array(
-            [
-                [
-                    _create_time_point(time - timedelta(hours=1)),
-                    _create_time_point(time),
-                ]
-                for time in times
-            ]
-        )
+        location_points = np.array([location for location in locations])
+        bounds = np.array([[location - 1, location] for location in locations])
+
         # Broadcast the times to a 2-dimensional grid that matches the diagnostic
         # data grid
-        time_points = np.broadcast_to(time_points, (5, 5))
+        location_points = np.broadcast_to(location_points, (5, 5))
         bounds = np.broadcast_to(bounds, (5, 5, 2))
-        # Create a 2-dimensional auxiliary time coordinate
-        self.time_aux_coord = iris.coords.AuxCoord(
-            time_points, "time", bounds=bounds, units=TIME_COORDS["time"].units
+        # Create a 2-dimensional auxiliary coordinate
+        self.location_aux_coord = iris.coords.AuxCoord(
+            location_points, long_name="location", bounds=bounds, units="degrees"
         )
 
         diagnostic_cube_2d_time = diagnostic_cube_yx.copy()
-        diagnostic_cube_2d_time.coord("time").rename("time_in_local_timezone")
-        diagnostic_cube_2d_time.add_aux_coord(self.time_aux_coord, data_dims=(0, 1))
+        diagnostic_cube_2d_time.add_aux_coord(self.location_aux_coord, data_dims=(0, 1))
 
         expected_indices = [[0, 0], [0, 0], [2, 2], [2, 2]]
-        points = [self.time_aux_coord.points[y, x] for y, x in expected_indices]
-        bounds = [self.time_aux_coord.bounds[y, x] for y, x in expected_indices]
-        self.expected_spot_time_coord = self.time_aux_coord.copy(
+        points = [self.location_aux_coord.points[y, x] for y, x in expected_indices]
+        bounds = [self.location_aux_coord.bounds[y, x] for y, x in expected_indices]
+        self.expected_spot_time_coord = self.location_aux_coord.copy(
```

Comment:
```suggestion
        self.expected_spot_aux_coord = self.location_aux_coord.copy(
```
And where it is used elsewhere in this file.

Reply:
updated everywhere